src/app/common/components/PipelineSummary.tsx

```diff
@@ -1,19 +1,124 @@
 import * as React from 'react';
+import { Flex, FlexItem, Text } from '@patternfly/react-core';
+import {
+  ResourcesAlmostEmptyIcon,
+  ResourcesAlmostFullIcon,
+  ResourcesFullIcon,
+} from '@patternfly/react-icons';
+import {
+  global_danger_color_100 as dangerColor,
+  global_disabled_color_200 as disabledColor,
+  global_info_color_100 as infoColor,
+  global_success_color_100 as successColor,
+} from '@patternfly/react-tokens';
 
-import { Text } from '@patternfly/react-core';
+import { IVMStatus } from '@app/queries/types';
+import { MigrationVMStepsType } from '@app/common/constants';
+import './PipelineSummary.css';
+
+interface IDashProps {
+  isReached: boolean;
+}
+
+const Dash: React.FunctionComponent<IDashProps> = ({ isReached }: IDashProps) => {
+  return (
+    <FlexItem alignSelf={{ default: 'alignSelfCenter' }}>
+      {isReached ? <div className="dash dashReached" /> : <div className="dash dashNotReached" />}
+    </FlexItem>
+  );
+};
+
+interface IChainProps {
+  /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Sorry for one last little thing.. can you make this `eslint-disable-next-line` instead of `eslint-disable`? As-is it disables that rule for the rest of the file or until an `eslint-enable` comment.